tests/fixtures/pokemon-footway.json

```diff
@@ -0,0 +1,95 @@
+{
+  "compareFunction": "pokemon-footway",
+  "fixtures": [
+    {
+      "description": "Looking for highway = footway with version = 1",
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "highway": "footway",
+          "osm:version": 1
+        },
+        "geometry": {
+          "type": "LineString",
+          "coordinates": [
+            [
+              77.57569670677185,
+              12.969890973519856
+            ],
+            [
+              77.57473111152648,
+              12.968186782548875
+            ],
+            [
+              77.57380843162537,
+              12.96650349043761
+            ]
+          ]
+        }
+      },
+      "oldVersion": null,
+      "expectedResult": {
+        "result:pokemonFootway": true,
+        "result:escalate": true
+      }
+    },
+    {
+      "description": "Looking for highway = footway with version != 1",
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "highway": "footway",
+          "osm:version": 4
+        },
+        "geometry": {
+          "type": "LineString",
+          "coordinates": [
+            [
+              77.57569670677185,
+              12.969890973519856
+            ],
+            [
+              77.57473111152648,
+              12.968186782548875
+            ],
+            [
+              77.57380843162537,
+              12.96650349043761
+            ]
+          ]
+        }
+      },
+      "oldVersion": null,
+      "expectedResult": {}
+    },
+    {
+      "description": "Looking for highway != footway with version != 1",
```

Comment:
This is the negative test.

I have a fixture for `highway=footway` and `version=1` here: https://github.com/mapbox/osm-compare/pull/65/files/3215494faf1f27796c8ae3c37d99fc9facc2b466#diff-19b695d540692f12666812d36ee9f53dR5